### DISCUSSION TITLE
Build beta versions for GitHub too

### DISCUSSION THREAD

**Discussion by j5k** [organicmaps/organicmaps#9437] [Q&A]
It seems like you are building google beta versions. As a person without google, but likely to test beta versions, is it possible to build the beta versions for GitHub too? 

In Obtainium I can include prereleases to keep track of updates.

**Comment by biodranik** [comment]
How do we avoid mixing (often experimental) alpha/beta versions with final/production releases on GitHub?

Is your issue that you can't download the beta apk without a Google account?

**Reply by j5k** [reply]
Yes, because I haven't a Google account, I can't test your beta versions. On Github you can tag a release as pre-release to mark a non production ready release: Point 10 from [1]: "Optionally, to notify users that the release is not ready for production and may be unstable, select This is a pre-release."

I'm not knowing Github enough but I think this should be automatable with CI's.

[1]: https://docs.github.com/en/repositories/releasing-projects-on-github/managing-releases-in-a-repository

**Reply by ghost** [reply]
What about other options for releasing beta's like F-Droid?

**Comment by VoxelPrismatic** [comment]
You can download the copy from GitHub Actions.

The beta is automatically pushed from there.
